suite-native/app/e2e/pageObjects/onboardingActions.ts

```diff
@@ -36,6 +36,13 @@ class OnOnboardingActions {
             );
         }
     }
+
+    async skipOnboarding() {
+        await waitFor(element(by.text('E2E:Skip onboarding')))
```

Comment:
Can we avoid duplicating the "id"? And preferably use `by.id`?

Reply:
Updated.